docs/timezone.md

```diff
@@ -50,8 +132,8 @@ Valid characters are ASCII letters, `.`, `-`, and `_`.
 For a list of IANA time zone names, see the current version of the [IANA time zone database](https://www.iana.org/time-zones).
 A convenient list is also available [on Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), although it might not reflect the latest official status.
 
-The string `timeZoneIdentifier` is canonicalized before being used to determine the time zone.
-For example, values like `+01` will be understood to mean `+01:00`, and capitalization will be corrected.
+The string `timeZoneIdentifier` is [canonicalized](#canonicalization-of-time-zone-identifiers) before being used to determine the time zone.
```

Comment:
This text should actually stay as it was, right? `+01` → `+01:00` and `utc` → `UTC`, but not `Asia/Calcutta` → `Asia/Kolkata`?

Reply:
Yes. The current docs have more currently-accurate text here. This change will be removed in my next rebase.